readme.md

```diff
@@ -1107,14 +1144,16 @@ In v22.2, use the following code:
    width: 100%;
 }
 ```
+
 [Return to the table of contents.](#thetableofcontents)
 
 
 ### DxComboBox 
 
-#### Change the height of a drop-down list
+#### Change the Height of a Drop-Down List
 
 In both v22.1 and v22.2, use the same razor code:
+
 ```cs
 <DxComboBox Data="@dataSource" @bind-Value="@Value" TextFieldName="Name" ValueFieldName="ID" DropDownCssClass="combo-dropdown">
 </DxComboBox>
```

Comment:
Please update line 1123 below (add a comma after "v22.1"):

`In v22.1, use the following code:`